.travis.yml

```diff
@@ -0,0 +1,49 @@
+# Travis CI (MIT License) configuration file
+# @link https://travis-ci.org/
+
+# Use new container based environment
+sudo: false
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+    # Aliased to a recent high PHP version
+    - 5.6
+    # Might as well test HHVM too
+    - hhvm
+    # Lowest currently possible in travis
+    - 5.2
```

Comment:
I'm pretty sure they reverted the removal of PHP 5.2 (under pressure of the WP community).

As we're currently only doing a lint and phpcs, running it on more versions would be wasting resources as the results won't change. If it breaks on either of those two versions, we need to look at the code, everything in between is caught by this.

Once we have unit tests, we should start running it on more versions, though still only lint and phpcs on highest and lowest.